wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java

```diff
@@ -61,5 +61,7 @@
 
   public static final String WS_MACHINE_NAME = "default";
 
+  public static final String SUPPORTED_ENVIRONMENTS = "supported_environment_recipe_types";
```

Comment:
Mb it would be better to use a simpler value like `supportedRecipes` or `supportedEnvironments`?

Reply:
Forget to apply the latest fix. Changed to `supported_environments`